tests/conftest.py

```diff
@@ -118,6 +134,43 @@ def labels():
     return blobs
 
 
+@pytest.fixture
+def prepare_cont_test_data():
```

Comment:
I'd use the full name for improving readability, so `continuous` in place of `cont`.